mammoth/documents.py

```diff
@@ -75,6 +75,8 @@ class Tab(Element):
 class Image(Element):
     alt_text = cobble.field()
     content_type = cobble.field()
+    content_width = cobble.field()
+    content_height = cobble.field()
```

Comment:
Any reason these couldn't just be `width` and `height`?

Reply:
The intention was to avoid confusion by making it clear we save the actual content info, and not the width/height as it is placed/transformed in the document. I imagined in the future there might be more attributes being saved (crop?) as this is quite valuable information for the image filter.